web3/utils/transactions.py

```diff
@@ -92,8 +92,19 @@ def get_required_transaction(web3, transaction_hash):
 
 
 def extract_valid_transaction_params(transaction_params):
-    return {key: transaction_params[key]
-            for key in VALID_TRANSACTION_PARAMS if key in transaction_params}
+    t_params = {key: transaction_params[key]
```

Comment:
*style nitpick*

We almost never abbreviate to a single letter. Some alternatives:
- `extracted_params`
- `cleaned_params`

Reply:
k.